active/NavigationView/TopPadding/TopPaddingMargins.md

```diff
@@ -0,0 +1,72 @@
+
+While this feels like the correct default behavior, we do not currently provide a mechanism for app developers to opt out of this automatic padding. App devs can `Window.Current.SetTitleBar` to their prefered UIElement, thus removing the hit test occlution by the transparent title bar across the entire window. And although this resolves the interaction problem, NavigationView's padding remains, providing now unwanted whitespace.
+
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+App developers have greater control over [customizing](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#navigation-view-customization) the margins and padding near the top of their instance of NavigationView. 
+
+# Navigation view customization
+<!-- Explanation and guidance that doesn't fit into the Examples
+section.  For example, see the Remarks for the MediaPlayerElement 
+(https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaPlayerElement#remarks). -->
+
+## Top whitespace
+Some apps choose to [customize their window's title bar](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar), potentially extending their app content into the title bar area. When NavigationView is the root element in apps that extend into the title bar, the control automatically adjusts the position of its interactive elements to prevent overlap with [the draggable region](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar#draggable-regions). If your app handles the draggable region of the title bar and you would prefer to have the back and menu buttons draw closer to the top of your app window, set `IsTitleBarPaddingEnabled` to False.
+
+````Xaml
+<muxc:NavigationView x:Name="NavView" IsTitleBarPaddingEnabled="False">
+````
+
+To further adjust the position of NavigationView's header area, override the *NavigationViewMinimalHeaderContentMargin* or *NavigationViewMinimalWithBackButtonHeaderContentMargin* XAML theme resources.  
+
+
+# API Notes
+<!-- Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. -->
+
+IsTitleBarPaddingEnabled - Specifies whether to automatically push NavigationView's interactive content down by the height of the title bar when the NavigationView is a root element of an app that extends into the title bar. The default value is True.
```

Comment:
Other proposed names for the property: 
IsTitleBarAutoPaddingEnabled
IsExtendedTitleBarPaddingEnabled

Reply:
I think I like IsTitleBarPaddingEnabled.  But I think we should fix the underlying problem so that this isn't typically necessary.  Would this then need to become a three state, like a nullable<bool>?

Reply:
@MikeHillberg - I'm sorry, I don't follow why this may need to be a nullable. NavigationView is already doing the right thing much of the time - getting out of the way of the title bar's back button and ensuring that its own back/hamburger buttons remain clickable. But NavigationView isn't currently allowing apps to get rid of this intelligent auto-padding, not even if the app author really, really wants to. So to me this feels like a binary choice: let NavigationView do its default thing or tell it to stop doing the default thing. What's the third state you think we might need?

Reply:
It's possible that Auto, NoPadding, and AlwayPadding.

Reply:
Yes, I was wondering if there should be TopPadding = on/off/auto.  But think it makes more sense the way you have it; AutoPadding = on/off.

Reply:
Proposed name: IsTitleBarAutoPaddingEnabled
Keep as bool